dpp/src/identity/state_transition/identity_credit_withdrawal_transition/mod.rs

```diff
@@ -0,0 +1,140 @@
+use serde::{Deserialize, Serialize};
+use serde_json::Value as JsonValue;
+use serde_repr::{Deserialize_repr, Serialize_repr};
+
+use crate::{
+    identity::KeyID,
+    prelude::Identifier,
+    state_transition::{
+        StateTransitionConvert, StateTransitionIdentitySigned, StateTransitionLike,
+        StateTransitionType,
+    },
+    util::json_value::{JsonValueExt, ReplaceWith},
+    ProtocolError,
+};
+
+use super::properties::{
+    PROPERTY_IDENTITY_ID, PROPERTY_OUTPUT, PROPERTY_OWNER_ID, PROPERTY_SIGNATURE,
+    PROPERTY_SIGNATURE_PUBLIC_KEY_ID,
+};
+
+pub mod apply_identity_credit_withdrawal_transition_factory;
+pub mod validation;
+
+pub const IDENTIFIER_FIELDS: [&str; 2] = [PROPERTY_IDENTITY_ID, PROPERTY_OWNER_ID];
+
+#[repr(u8)]
+#[derive(Serialize_repr, Deserialize_repr, PartialEq, Eq, Clone, Copy, Debug)]
+pub enum Pooling {
+    Never = 0,
+    IfAvailable = 1,
```

Comment:
We support pooling never only atm

Reply:
Done, using json schema (allowing only 0, for now)